pints/functionaltests/differential_evolution.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+
+from __future__ import division
+
+import pints
+
+from ._problems import (RunMcmcMethodOnTwoDimGaussian,
+                        RunMcmcMethodOnBanana,
+                        RunMcmcMethodOnCorrelatedGaussian,
+                        RunMcmcMethodOnAnnulus)
+
+
+def test_differential_evolution_on_two_dim_gaussian(n_iterations=None):
```

Comment:
Would it be a good idea to name these methods as `two_dim_gaussian` etc. so that we can write `pints.functionaltests.differential_evolution.two_dim_gaussian()` ?